tests/Tasks/UpdatePackageInfoTest.php

```diff
@@ -48,7 +48,13 @@ public function testPackagesAreAddedCorrectly()
     {
         $task = UpdatePackageInfoTask::create();
 
-        $frameworkVersion = VersionProvider::singleton()->getModuleVersion('silverstripe/framework');
+        /** @var VersionProvider $versionProvider */
+        $versionProvider = $this->getMockBuilder(VersionProvider::class)
```

Comment:
Fixes unit test when doing a recipe-sink CI run with a forked version of framework

Reply:
Didn't you already do that? I feel sure I remember you doing that before.

Reply:
Yeah actually I did, though it was for CMS 6, part of the phpunit 11 work - https://github.com/bringyourownideas/silverstripe-maintenance/commit/0dc17981b1aa44b8aa63965582f8bbc16587fae6

This is still worth merging as is I think rather thank backporting, because I'm not sure if the try/finally block is required

We can update the cms 6 version to include the try/finally block on merge-up